lms/envs/common.py

```diff
@@ -3533,6 +3533,9 @@ def _make_locale_paths(settings):
 COLARAZ_BLOCKED_FULL_PATHS = ['/', '/login']
 COLARAZ_ALLOWED_SUB_PATHS = ['/api/commerce']
 
+# To disable account verification for colaraz user
+FEATURES['ENABLE_AUTOMATIC_ACCOUNT_VERIFICATION'] = True
```

Comment:
shouldn't we set this from "lms.env.json" ?

Reply:
Yep, we can but it's the default value.

Reply:
But we should add it there too.

Reply:
agree with @danialmalik, I think we should add it there too so that it can be overwritten without changing the code in future.

Reply:
WE DON'T NEED TO DO THAT AS IT WILL BE AUTOMATICALLY POPULATED IN production.py

![image](https://user-images.githubusercontent.com/42294172/88170071-7dda9a00-cc36-11ea-97e3-ffbfb372cda9.png)